ui/app/components/clients/current.js

```diff
@@ -11,9 +12,20 @@ export default class Current extends Component {
   });
   @tracked selectedNamespace = null;
 
-  // TODO CMB get from model
   get upgradeDate() {
-    return this.args.upgradeDate || null;
+    let keyInfoObject = this.args.model.versionHistory.key_info;
+    if (!keyInfoObject) {
+      return false;
+    }
+    let firstKey = Object.keys(keyInfoObject); // key name here will change, but we want the first one as this indicates when they upgraded to 1.9 or above
+    let versionDate = new Date(keyInfoObject[firstKey].timestamp_installed);
+    // compare against this month and this year to show message or not.
+    let compare = compareAsc(versionDate, startOfMonth(new Date()));
```

Comment:
is there a reason you chose `compareAsc` over `isAfter`? The latter feels more readable, and would turn it into a one liner

`return isAfter(versionDate, startOfMonth(newDate())`

Reply:
nope, just didn't know that one existed. Searched the docs and the compareAsc came first. I like isAfter. I'll amend.